package.json

```diff
@@ -16,7 +16,7 @@
     "test": "npm run lint && npm run jest; exit 0;",
     "build": "NODE_ENV=production webpack --mode production",
     "build:profile": "NODE_ENV=production webpack --mode production --profile",
-    "build:watch": "NODE_ENV=production webpack --mode production --watch",
+    "build:watch": "NODE_ENV=production webpack --mode development --watch",
```

Comment:
@janwidmer Patric said you might know what the `NODE_ENV=production` part here was necessary for. Do you have any ideas? Or should we get rid of it too?

Reply:
We investigated this. I think the variable was added, because `eslint-import-resolver-webpack` executes the webpack config excessively but does not emit the previous environment variables. But since the environment has no impact on the resolver it should be save to remove this.

- I will remove the additional `NODE_ENV` and only use `--mode`
- I will remove `mode` and `watch` from the exported config, since the mode is defined by the script (which has precedence)